src/main/kotlin/com/jetbrains/snakecharm/lang/SnakemakeNames.kt

```diff
@@ -53,6 +54,7 @@ object SnakemakeNames {
     const val SECTION_RUN = "run"
     const val SECTION_CACHE = "cache" // >= 5.12.0
     const val SECTION_CONTAINER = "container"
+    const val SECTION_CONTAINERIZED = "containerized"
```

Comment:
could you please add a comment in which version the feature was introduced, like for SECTION_CACHER, etc sections here.

Reply:
https://github.com/JetBrains-Research/snakecharm/pull/381/commits/f0cbee9bad59cc1aa09c2d78a052ecb2880ba48f